src/components/partner/partner.resolver.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  Args,
+  Mutation,
+  Parent,
+  Query,
+  ResolveField,
+  Resolver,
+} from '@nestjs/graphql';
+import { IdArg, ISession, Session } from '../../common';
+import { OrganizationService, SecuredOrganization } from '../organization';
+import {
+  CreatePartnerInput,
+  CreatePartnerOutput,
+  Partner,
+  PartnerListInput,
+  PartnerListOutput,
+  UpdatePartnerInput,
+  UpdatePartnerOutput,
+} from './dto';
+import { PartnerService } from './partner.service';
+
+@Resolver(Partner)
+export class PartnerResolver {
+  constructor(
+    private readonly partnerService: PartnerService,
+    private readonly orgService: OrganizationService
+  ) {}
+
+  @Mutation(() => CreatePartnerOutput, {
```

Comment:
Would you mind moving this after the query & field resolvers so that all the mutations are grouped together?